test/integration/targets/bigip_device_info/tasks/issue-01563.yaml

```diff
@@ -0,0 +1,94 @@
+---
+
+- name: Issue 01563 - Include issue variables
+  include_vars:
+    file: issue-01563.yaml
+
+- name: Remove VLAN
```

Comment:
do we need to remove vlan? next task will either create vlan or ignore. Maybe add all this into setup-01563.yaml and import.

Reply:
Good point. The reference I took was probably assuming that the VLAN needs to be removed and re-added.